airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -520,6 +520,24 @@ def test_oauth_flow_parameters(self, actual_connector_spec: ConnectorSpecificati
 
         diff = paths_to_validate - set(get_expected_schema_structure(spec_schema))
         assert diff == set(), f"Specified oauth fields are missed from spec schema: {diff}"
+        """
+        OAuth is default check.
+        If credentials do have oneOf: we check that the OAuth is listed at first.
+        If there is no oneOf and Oauth: OAuth is only option to authenticate the source and no check is needed.
+        """
+        if advanced_auth.predicate_key:
+            credentials = advanced_auth.predicate_key[0]
+            one_of_default_method = None
+            try:
+                one_of_default_method = dpath.util.get(spec_schema, f"/**/{credentials}/oneOf/0")
+            except KeyError:  # Key Error when oneOf is not in credentials object
+                pass
```

Comment:
It would have been nice to keep a trace of this using `pytest.skip`. However, since this test is testing many things, it wouldn't be true that we are skipping the whole test. Is there a reason not to create a new test for that?